src/content/docs/design/content-customization/how-translation-works.mdx

```diff
@@ -0,0 +1,50 @@
+---
+page_id: d5ddfbe2-6608-45d4-8c46-44be93690c9d
+title: Page content translation
+sidebar:
+  order: 3
+relatedArticles:
+  - b3a309c6-af3c-4155-b39e-7e1f6e83721f
+  - 2aa551b8-06c0-4947-bd4b-d643c77ed224
+
+---
+
+Here’s how translation is handled when you change page content.
+
+### Language detection
+
+Language detection is automatically handled for you and Kinde gives display priority to languages as follows:
+
+1. The language supplied in the auth url ^
+2. The preferred language selected in the user’s browser ^
+3. The default language you have selected in Kinde
+
+^ If you have chosen not to support a language, we will try the next available option.
+
+Through language matching, Kinde automatically works out the closest 
+base language to the requested language and will use this if supported 
+by your app.
+
+After the priority list outlined above Kinde will fall back to US English as a last resort, if a translation is not available.
+
+### Widget translations
+
+Translations inside of the Kinde widget are taken care of by Kinde. We will grab the copy from the relevant route and language from your `Page content` settings inside of Kinde and apply them to the widget. 
+
+### Page translations
```

Comment:
The widget/page sections are really specific to custom UI. If you're not using custom UI you probably won't make much sense of it